cmd/analyzer/main.go

```diff
@@ -36,6 +36,10 @@ var (
 )
 
 func init() {
+	if err := cmd.VerifyCompatibility(); err != nil {
```

Comment:
I think `analzyer -version` should still work even if the API isn't compatible. It would help folks figure out why the API isn't compatible.

Reply:
same for the rest of the phases

Reply:
given the difficulty entailed in implementing this suggestion, we can punt on it for now